apps/sensenet/src/components/dialogs/permission-editor-dialog.tsx

```diff
@@ -337,7 +337,10 @@ export function PermissionEditorDialog(props: PermissionEditorDialogProps) {
             <Button
               aria-label={localization.forms.cancel}
               className={globalClasses.cancelButton}
-              onClick={closeLastDialog}>
+              onClick={() => {
+                closeLastDialog()
+                props.callBackFunction?.()
```

Comment:
it is not the best solution, is it possible to refesh right after new entry has added or here, but only in that case?